README.md

```diff
@@ -1,66 +1,64 @@
-# vue-webpack-boilerplate
+<p align="center">
+  <img src="./w3tec-logo.png" alt="w3tec" width="400" />
+</p>
 
-> A full-featured Webpack setup with hot-reload, lint-on-save, unit testing & css extraction.
+<h1 align="center">Vue Typescript Template</h1>
 
-> This template is Vue 2.0 compatible. For Vue 1.x use this command: `vue init webpack#1.0 my-project`
+<p align="center">
+  <b>A full featured <a href="https://github.com/vuejs/vue-cli">vue-cli</a> TypeScript and Webpack template for Vue.js 2.0</b>
+  </br></br>
+  <span>
+    Inspired by official <a href="https://github.com/vuejs-templates/webpack">Vue.js Webpack template</a>
+  </span>
+  </br>
+  <sub>
+    Made with ❤️ by <a href="https://github.com/w3tecch">w3tech</a>, <a href="https://twitter.com/rkleger">Robert Kleger</a> and <a href="https://github.com/w3tecch/vue-typescript-template/graphs/contributors">contributors</a>
```

Comment:
Please remove names, as we start to remove the names in any other boilerplate too.

Reply:
Done